Lib/uuid.py

```diff
@@ -581,6 +589,18 @@ def _netbios_getnode():
     _generate_time_safe = getattr(_uuid, "generate_time_safe", None)
     _UuidCreate = getattr(_uuid, "UuidCreate", None)
     _has_uuid_generate_time_safe = _uuid.has_uuid_generate_time_safe
+
+    if sys.platform == "darwin" and _generate_time_safe is not None:
```

Comment:
I'm not 100% sure about this test, and in particular not about the test for "darwin".  

There are people running Linux on Apple hardware and I don't know if those could be affected by this issue, in particular if there is a driver that exposes a ethernet-like interface to the Touch Bar hardware.